### DISCUSSION TITLE
Make it possible to organize the discussion board by most upvoted ideas

### DISCUSSION THREAD

**Discussion by Leonidas-ai** [FakeFishGames/Barotrauma#5736] [Ideas and suggestions]
I see no way to do this. Are we supposed to flip through millions of ideas to find the good ones?

**Comment by Hinny** [comment]
Just klick the button named "Top:all" and select "all". Not the most intuitive interface but it should more or less do the trick. 

If you have further problems with this I suggest you file a GitHub issue https://support.github.com/contact/feedback since it has nothing to do with the Barotrauma development.

**Reply by Leonidas-ai** [reply]
Didn't solve the problem. Get the hell out of here.

**Reply by Hinny** [reply]
That is very strange, it does for me.

Why are you so angry? I am trying to help here.
